src/modules/python/pbs/v1/_svr_types.py

```diff
@@ -1365,3 +1365,136 @@ def __next__(self):
                 return _pbs_v1.iter_nextfunc(self, 0, self.obj_name,
                                              self.filter1, self.filter2)
 #: C(pbs_iter)
+
+#:------------------------------------------------------------------------
+#                  SERVER ATTRIBUTE TYPE
+#:-------------------------------------------------------------------------
+class _server_attribute:
+    """
+    This represents a external form of attributes..
+    """
+    attributes = PbsReadOnlyDescriptor('attributes', {})
+    _attributes_hook_set = {}
+    def __init__(self, name, resource, value, op, flags):
+        self.name = name
+        self.resource = resource
+        self.value = value
+        self.op = op
+        self.flags = flags
+        self.sisters = []
+    #: m(__init__)
+
+    def __str__(self):
+        return "%s:%s:%s:%s:%s" % self.tup()
```

Comment:
self.tup() returns 6 tuples but you only have 5 %s, it should be 6 %s. I realized this hen I tried the following management hook: 
for a in pbs.event().management..attribs:
    pbs.logmsg(pbs.LOG_DEBUG, "%s" % (a))

I got an exception:
3/13/2020 02:17:56.296640;0001;Server@corretja;Svr;Server@corretja;PBS server internal error (15011) in Error evaluating Python script, <class 'TypeError'>
03/13/2020 02:17:56.296644;0001;Server@corretja;Svr;Server@corretja;PBS server internal error (15011) in Error evaluating Python script, not enough arguments for format string

Better yet, for readability, just do:
def __str__(self):
        return "name=%s:resource=%s:value=%s:op=%s:flags=%s:sisters=%s" % self.tup()

So the output is more readable:
3/13/2020 02:24:31.334731;0006;Server@corretja;Hook;Server@corretja;a=name=enabled:resource=None:value=true:op=0:flags=0:sisters=[]